### DISCUSSION TITLE
Asking for Bluetooth permission in A13

### DISCUSSION THREAD

**Discussion by Robbo5ooo** [AppGround-io/bluetooth-keyboard-and-mouse-support#291] [Bugs & Problems]
Have been using the app for some time with no issues. However I've just recently updated to Android 13 and now, every time I start the app, it is asking for permission to disable, then enable Bluetooth. The app has Bluetooth permissions enabled, so it's very annoying to continually ask again. This has happened on both my phone and tablet after updating both from LineageOS 19.1 to 20.0.
Obviously something has changed in A13 to cause this around Bluetooth permissions and security.
Don't know if something new needs declaring in the manifest or if the issue is elsewhere.
The app isn't listed in the 'Modify System Settings' special app access settings. Should it be listed here?

**Comment by AppGround-io** [comment]
In Android 13, the API to enable or disable Bluetooth is no longer supported for regular apps. As a result, you'll have to manage Bluetooth settings manually. Unfortunately, there isn't a workaround for this issue.

**Comment by Robbo5ooo** [comment]
OK, thank you for the update.
Wasn't sure what the new A13 policies were. Was hoping it was just ab new elevation of permissions and required an extra declaration in the manifest. I'll just have to live with OKing the permission to enable and disable Bluetooth.

**Comment by mortee** [comment]
I have the same issue after installing the app on my Galaxy A52, just worse: it goes to "checking device capabilities", then asks to turn off BT, then back on again, then shows the list of devices and the "set up device" button for a second, then immediately restarts the whole cycle before I can do anything. Completely unusable in current form.

**Reply by mortee** [reply]
Correction: it only did this when I installed it in my "work" profile. In my normal profile, it actually works really decently, thank you!
